server/app/services/triplers.js

```diff
@@ -86,6 +88,11 @@ async function findRecentlyConfirmedTriplers() {
 async function confirmTripler(triplerId) {
   let tripler = await neode.first("Tripler", "id", triplerId)
   let ambassador = tripler.get("claimed")
+  if(ambassador.get('hs_id'))
```

Comment:
Let's make it clear that the function in `ambassadorsSvc` is responsible for checking whether `hs_id` is populated, and then remove the check here.

Reply:
added context